ledger/src/blockstore.rs

```diff
@@ -2604,6 +2607,24 @@ impl Blockstore {
         self.perf_samples_cf.put(index, perf_sample)
     }
 
+    // program_cost table (if possible), it'd break a bit of pattern for ReplayStage to
+    // write. Can CF be gloabl - not tethed to slot?
+    pub fn read_program_cost(&self) -> Result<Vec<(Pubkey, u64)>> {
```

Comment:
From the PR notes,
```
replay_stage writes current cost table to blockstore whenever it replayed active banks, so the latest information are persisted;
...
validator restores cost table into cost_model during startup.
```
To confirm then, it doesn't sound like there is ever a use case to need to read a single program cost out, right? The cost_model is kept up to date in memory (after initialization), and updates are just pushed to the blockstore for persistence?

Regardless, change the function name to be plural to `read_program_costs()`

Reply:
Yes, and changed the function name